### PR TITLE
Explicitly use debian buster for apollo/server images

### PR DESCRIPTION
**Pull request by madkinsz**
## Summary

- Upgrades the `apollo` image to use `buster` instead of `stretch` Debian. 
- Changes the `server` image to explicitly use `buster` but should not change anything as it was implicit before

## Importance

- Consistent base OS
- Security patches (see #198) 

## Checklist

This PR:

- ~[ ] adds new tests (if appropriate)~
- [x] adds a change file in the `changes/` directory (if appropriate)

### CODE REVIEW

File: services/apollo/healthcheck.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+var http = require("http");
```
Comment (cicdw) [inline services/apollo/healthcheck.js:1]: jw how is this file related to the image changes?

Comment (madkinsz) [reply services/apollo/healthcheck.js:1]: Whoops! That's not supposed to be in here. I'll remove this :)

Comment (madkinsz) [reply services/apollo/healthcheck.js]: I was exploring using node for a healthcheck instead of curl since it's less of a security risk

Comment (cicdw) [reply services/apollo/healthcheck.js]: nice - glad i asked haha

**Review by cicdw — APPROVED** [review]
